llvm/test/CodeGen/LoongArch/addrspacecast.ll

```diff
@@ -0,0 +1,47 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=loongarch32 -verify-machineinstrs < %s | FileCheck %s --check-prefix=LA32
```

Comment:
Canonicalized options use two dashes (`--mtriple=` and `--verify-machineinstrs`) from `llc --help-hidden`'s prompt.

Reply:
Thanks.